src/autocapture.ts

```diff
@@ -243,25 +244,13 @@ export class Autocapture {
         }
     }
 
-    private _getEventTarget(e: Event): Element | null {
```

Comment:
this is useful dark magic, so we lift it out of the class so that heatmaps can use it